app/models/user.rb

```diff
@@ -51,10 +56,12 @@ class User < ActiveRecord::Base
     ],
   ]
 
+
   INVALID_EMAIL = 'invalid_email'
 
   devise  :openid_authenticatable, :trackable
 
+
```

Comment:
"Kannst du die leeren Zeilen hier und in .gitignore, users_controller.rb und am Ende von work_days/index.html.erb_spec.rb und routes.rb bitte entfernen?"